control/nlsys.py

```diff
@@ -2500,3 +2579,39 @@ def _convert_static_iosystem(sys):
         return NonlinearIOSystem(
             None, lambda t, x, u, params: sys @ u,
             outputs=sys.shape[0], inputs=sys.shape[1])
+
+def connection_table(sys, show_names=False, column_width=32):
+    """Print table of connections inside an interconnected system model.
+
+    Intended primarily for :class:`InterconnectedSystems` that have been
+    connected implicitly using signal names.
+
+    Parameters
+    ----------
+    sys : :class:`InterconnectedSystem`
+        Interconnected system object
+    show_names : bool (optional)
+        Instead of printing out the system number, print out the name of
+        each system. Default is False because system name is not usually
+        specified when performing implicit interconnection using
+        :func:`interconnect`.
+    column_width : int (optional)
+        Character width of printed columns
+
+
+    Examples
+    --------
+    >>> P = ct.ss(1,1,1,0, inputs='u', outputs='y', name='P')
+    >>> C = ct.tf(10, [.1, 1], inputs='e', outputs='u', name='C')
+    >>> L = ct.interconnect([C, P], inputs='e', outputs='y')
+    >>> L.connection_table(show_names=True) # doctest: +SKIP
+    signal    | source                  | destination
+    --------------------------------------------------------------
+    e         | input                   | C
+    u         | C                       | P
+    y         | P                       | output
+    """
+    assert isinstance(sys, InterconnectedSystem), "system must be"\
+        "an InterconnectedSystem."
+
+    sys.connection_table(show_names=show_names, column_width=column_width)
```

Comment:
Not something that we need to fix in this PR, but we should eventually adopt some mechanism for creating duplicate docstrings from a single primary docstring (see, for example, the discussion [here](https://softwareengineering.stackexchange.com/questions/386755/sharing-docstrings-between-similar-functions)).